### PR TITLE
Implement pundit

### PR DESCRIPTION
**Pull request by RachalCassity**
Implemented pundit. 

Pundit will need to be added to the application_controller_spec and sentry logging spec when those are implemented.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -14,6 +14,7 @@ gem 'flipper-ui',        '~> 0.23' # UI for the Flipper gem
 gem 'jbuilder',          '~> 2.11' # Build JSON APIs with ease [https://github.com/rails/jbuilder]
 gem 'pg',                '~> 1.1'  # Use postgresql as the database for Active Record
 gem 'puma',              '~> 5.0'  # Use the Puma web server [https://github.com/puma/puma]
+gem 'pundit'
```
Comment (LindseySaari) [inline Gemfile:17]: I believe we're using the pattern of pessimistically locking the gems in the Gemfile to the latest minor version. Also if you could, add a small comment summarizing what the gem does

File: brakeman.json [hunk 2]
```diff
@@ -0,0 +1,103 @@
+{
+  "scan_info": {
+    "app_path": "/Users/rachalcassity/va/platform-console",
```
Comment (ericboehs) [inline brakeman.json:3]: I don't think you mean to commit brakeman.json and rubocop.json. Could you add them to your global gitignore?

File: .rubocop.yml [hunk 3]
```diff
@@ -5,3 +5,6 @@ AllCops:
 
 Metrics/MethodLength:
   Max: 15
+
+Style/Documentation:
```
Comment (ericboehs) [inline .rubocop.yml:9]: I'm trying to go with the default Rubocops. I've been trying it out and so far it wasn't too bad to keep this documentation cop enabled.

Thoughts?

Comment (RachalCassity) [reply .rubocop.yml:9]: I will remove these this change.

**Pull request closed without merging**